build.sbt

```diff
@@ -180,6 +178,25 @@ lazy val ratatoolShapeless = project
     ratatoolSampling
   )
 
+lazy val ratatoolExtras = project
```

Comment:
Do we need to override some setting here so it does not get `pack`ed?

Reply:
(we discussed offline and this is fine as is)